src/main/scala/gw/cda/api/outputwriter/JdbcOutputWriter.scala

```diff
@@ -29,22 +22,36 @@ import org.apache.spark.sql.types.LongType
 import org.apache.spark.sql.types.ShortType
 import org.apache.spark.sql.types.StringType
 import org.apache.spark.sql.types.TimestampType
-import org.apache.spark.sql.{DataFrame, functions => sqlfun}
 import org.apache.spark.sql.Row
 import org.apache.spark.sql.SparkSession
 import org.apache.spark.sql.functions.col
 import org.apache.spark.sql.jdbc.JdbcDialects
 import org.apache.spark.sql.types.StructType
 
+import java.io.File
+import java.io.IOException
+import java.net.URI
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.sql.Connection
+import java.sql.DriverManager
+import java.sql.PreparedStatement
+import java.sql.SQLException
+import java.util.Locale
+
 private[outputwriter] class JdbcOutputWriter(override val outputPath: String, override val includeColumnNames: Boolean,
                                              override val saveAsSingleFile: Boolean, override val saveIntoTimestampDirectory: Boolean,
                                              override val clientConfig: ClientConfig) extends OutputWriter {
 
   private[outputwriter] val configLargeTextFields: Set[String] = Option(clientConfig.outputSettings.largeTextFields).getOrElse("").replace(" ", "").split(",").toSet
 
   override def validate(): Unit = {
-    if (!Files.isDirectory(Paths.get(outputPath))) {
-      throw new IOException(s"$outputPath is either not a local directory or doesn't exist")
+    val pathExists = new URI(outputPath).getScheme match {
```

Comment:
ditto previous comment about scheme

Reply:
Hi, Would you be able to share what spark, Hadoop and Scala version you used? We are trying to run the code in local mode with Spark 3.0.3 that comes with Hadoop2,7, but failing on version mismatch issues. Your input is highly appreciated.